### PR TITLE
Restaurant Page

### PR DESCRIPTION
**Pull request by ZawadiSumba66**
A restaurant page that showcases tabbed content for the menu, contact, and about tabs

### CODE REVIEW

File: src/menu.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import pizza from './pizza.jpg';
+
+const menuList = (text) => ({ text });
+// const menuImage = (image) => ({ image });
```
Comment (MahmoudBakr23) [inline src/menu.js]: Remove this line.

File: src/menu.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+const listOne = menuList('Burger..............KSH. 700');
+const listTwo = menuList('Kiwi Mocktail..............KSH. 400');
+const listThree = menuList('Taandori chicken wrap..............KSH. 600');
+const listFour = menuList('Java Icecream..............KSH. 250');
+const listFive = menuList('Java Granola bars ..............KSH. 200');
+const listSix = menuList('Coffe and bakery combo..............KSH 400');
+// const listImage = menuImage(pizza)
```
Comment (MahmoudBakr23) [inline src/menu.js]: Remove this line.

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
# **Changes Required 🛑♻**

<hr>

Hello there :)
Great work and effort you put into this good-looking app 👍 You've done well setting up the Linter, Webpack, dist directory, gitignored node_modules folder, DOM manipulation code separated from the logic code, and descriptive PR and commit messages. 💯 🥇 
However, there is always room for improvement. Please consider the following:-

## REQUIRED

- [ ] Kindly remove any commented-out code so it stays clean. See my comments below!
- [ ] In your README file, please make it clear to the user how to run the app locally. Ex: ```npm install``` ```npm run build``` with explaining what they do.

## OPTIONAL

- To start a live server locally, please add this script to your JSON package file
```"start": "webpack-dev-server --mode development --open"```
- It'd be great if you create a new folder called assets/images and put all the images inside of it.
- It'd be nice if you include a screenshot or two of your app in the README file.

<hr>

That's it!
Good luck and happy coding 👨‍💻💻

<hr>

Please feel free to contact me on Slack with the name "Mahmoud Bakr" if you had any questions.

**Review by Temmarie — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi @ZawadiSumba66  ! :wave:

#### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: :tada:

#### You may merge, good luck with your next projects! :rocket:
#### Happy Coding! :+1:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Pull request merged**